compiler/analyzer.ml

```diff
@@ -53,6 +59,7 @@ let rec str_of_type = function
   | Void -> "Void"
   | List(l) -> sprintf "List[%s]" (str_of_type l)
   | Func(f) -> str_of_func f
+  | Dist(d) -> "Dist" (* Need to edit *)
```

Comment:
@lillyfwang @alexandramedway - dist constraining is tentatively complete in the dist_constraining branch. However, I have neither tested the code nor written any fail/pass tests for it. I will do this tomorrow.